deploy/openshift/kiali-secrets.yaml

```diff
@@ -0,0 +1 @@
+../kubernetes/kiali-secrets.yaml
```

Comment:
don't make this a symlink - we need the actual text here. Because otherwise when someone is on OpenShift and we give them the quick command line instructions to install (see https://kiali.org), I don't think this would work:

```
curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-secrets.yaml | \
   VERSION_LABEL=master envsubst | oc create -n istio-system -f -
```

Unless GitHub is actually smart enough to produce the actual raw text from a symlink using that raw.githubusercontent.com URL.

Reply:
The raw github URL does not provide the text - if I go to https://raw.githubusercontent.com/israel-hdez/swscore/kiali-855/deploy/openshift/kiali-secrets.yaml (the raw content for this symlink in @israel-hdez branch for this PR) all I get is the pointer to the file (`../kubernetes/kiali-secrets.yaml`) so this does need to be fixed before we can merge.

Reply:
Fixed, now it's a real file.